### PR TITLE
feat: Add Rucio S3 support Fellow project

### PR DESCRIPTION
**Pull request by matthewfeickert**
Add IRIS-HEP Fellow project description for 'Rucio-S3-compatible access interface for analysis facilities'.

Tagging @lukasheinrich and @mlassnig to sign off on this as this is originally their idea from GSoC 2022 and they're listed as co-mentors with me.

Tagging @oshadura as this would be a good Ukrainian Fellows project. :+1:

### CODE REVIEW

**Comment by matthewfeickert** [comment]
This PR is fine but c.f. Issue #6 for why the CI is failing.

File: projects/rucio-s3.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+name: Rucio-S3-compatible access interface for analysis facilities
+postdate: 2023-05-07
+categories:
+  - Analysis tools
+  - Open science
+durations:
+  - 3 months
+skillset:
+  - Python
+  - C++
```
Comment (matthewfeickert) [inline projects/rucio-s3.yml:11]:
```suggestion
  - Docker
```

As the original proposal https://hepsoftwarefoundation.org/gsoc/2022/proposal_Rucio-S3.html mentions Docker but not C++ I'm going to swap these.

**Comment by oshadura** [comment]
LGTM! @davidlange6 would it be possible to merge it?

**Comment by lukasheinrich** [comment]
tagging @mlassnig

File: projects/rucio-s3.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - IRIS-HEP
+experiments:
+  - ATLAS
+program:
+  - IRIS-HEP fellow
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Implement a REANA workflow for dark matter searches at RCFM
```
Comment (lukasheinrich) [inline projects/rucio-s3.yml:24]: this seems still off

Comment (matthewfeickert) [reply projects/rucio-s3.yml:24]: oh whoops! Indeed that was a copy-paste gone wrong. Good catch @lukasheinrich.

File: projects/rucio-s3.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  - IRIS-HEP
+experiments:
+  - ATLAS
+program:
+  - IRIS-HEP fellow
+location:
+  - Remote
+commitment:
+  - Full time
+shortdescription: Add S3 compatible access interface to Rucio
```
Comment (matthewfeickert) [inline projects/rucio-s3.yml:24]: @lukasheinrich Look better now that I remembered to update it to the correct project?

**Comment by davidlange6** [comment]
ok, I think now I just need to convince the ci to use the most recent commit in the repo.. sorry for the noise.

**Comment by matthewfeickert** [comment]
@davidlange6 I've rebased so it is passing CI but c.f. also PR #33.

**Pull request merged**